Tools/Scripts/filter-test-logs

```diff
@@ -94,18 +92,23 @@ def filter_webdriver_tests(log_file_name):
             if 'tests ran as expected' in line:
                 print_line = True
                 print(line, end='')
+    return print_line
 
 def main():
     args = parser()
+    log_file = os.path.abspath(f'{args.output}')
     if args.filter_type == 'jsc':
-        filter_jsc_tests(args.output)
+        printed_results = filter_jsc_tests(log_file)
     elif args.filter_type == 'layout':
-        filter_layout_tests(args.output)
+        printed_results = filter_layout_tests(log_file)
     elif args.filter_type == 'scan-build':
-        filter_scan_build(args.output)
+        printed_results = filter_scan_build(log_file)
     elif args.filter_type == 'webdriver':
-        filter_webdriver_tests(args.output)
-
+        printed_results = filter_webdriver_tests(args.output)
+    if not printed_results:
+        with open(log_file, 'r') as raw_output:
+            sys.stdout.write(raw_output.read())
```

Comment:
Will it print EVERYTHING (which might be hundreds of thousands of log lines)? 
The reason we moved huge logs away from buildbot and to S3 is that they significantly affect the buildbot performance. We don't want to print out hundreds of thousands of log lines. 
It's ok to print few hundred lines of logs needed for debugging. 

cc @briannafan @gsnedders

Reply:
I thought this would be a rare enough occurrence (it only happens when the script throws an exception that is not caught, not when e.g. tests error out or time out) but yah, I guess the last few hundred lines would work for diagnosing. Let's make it 500 to be on the safe side?

Reply:
Yeah, 500 (or even 1000) lines is fine.